scripts/Imaging/IMAT/tomo_reconstruct.py

```diff
@@ -54,10 +54,10 @@
 import os
 from os import path
 # So insert in the path the directory that contains this file
-sys.path.insert(0, os.path.split(path.dirname(__file__))[0]) # noqa
+sys.path.insert(0, os.path.split(path.dirname(__file__))[0])  # noqa
```

Comment:
WRGT sys.path.insert: http://stackoverflow.com/a/10097543

Additionally what are we appending to the system path - personally this line scares me, isn't this what virtual environments are for?